website/src/documentation/patterns/side-inputs.md

```diff
@@ -50,25 +50,24 @@ For instance, the following code sample uses a `Map` to create a `DoFn`. The `Ma
 
 ## Slowly updating side input using windowing
 
-You can read side input pcollection periodically into distinct windows.
-Later, when you apply side input to your main input, windows will be matched automatically 1:1.
-This way, you can guarantee side input consistency on the duration of the single window.
+You can read side input data periodically into distinct PCollection windows.
```

Comment:
In Beam, we have IO.Read transforms that do read source data. Later data read can be assigned to specific window.